File: C#/GLA_APP/Legislature.cs
// Legislature.cs
using System;

public class Legislature : Gla, IUtility
{
    private string Speaker = "Shri. Ramesh Tawadkar";
    private string DutySpeaker = "Shri. Joshua De Souza";
    private string OppLeader = "Shri. Yuri Alemao";
    private string[] MLA_Array = 
    {
        "SHRI. JIT AROLKAR",
    "SHRI. PRAVIN ARLEKAR",
    "SHRI. CHANDRAKANT SHETYE",
    "SHRI. NILKANTH HALARNKAR",
    "SHRI. JOSHUA DE SOUZA",
    "SMT. DELILAH LOBO",
    "SHRI. KEDAR NAIK",
    "SHRI. MICHAEL LOBO",
    "SHRI. ROHAN KHAUNTE",
    "SHRI. CARLOS FERREIRA",
    "SHRI. ATANASIO MONSERRATE",
    "SMT. JENNIFER MONSERRATE",
    "SHRI. RODOLFO FERNANDES",
    "SHRI. VIRESH BORKAR",
    "SHRI. RAJESH FALDESSAI",
    "SHRI. PREMENDRA SHET",
    "SHRI. PRAMOD SAWANT",
    "SMT. DEVIYA RANE",
    "SHRI. VISHWAJIT RANE",
    "SHRI. GOVIND GAUDE",
    "SHRI. RAVI NAIK",
    "SHRI. SUBHASH SHIRODKAR",
    "SHRI. RAMKRISHNA ALIAS SUDIN DHAVALIKAR",
    "SHRI. SANKALP AMONKAR",
    "SHRI. KRISHNA SALKAR",
    "SHRI. MAUVIN GODINHO",
    "SHRI. ANTONIO VAS",
    "SHRI. ALEIXO SEQUEIRA",
    "SHRI. ALEIXO LOURENCO",
    "SHRI. VIJAI SARDESAI",
    "SHRI. DIGAMBAR KAMAT",
    "SHRI. VENZY VIEGAS",
    "SHRI. ULHAS TUENKAR",
    "SHRI. YURI ALEMAO",
    "SHRI. CRUZ SILVA",
    "SHRI. ALTONE D’COSTA",
    "SHRI. NILESH CABRAL",
    "SHRI. GANESH GAONKAR",
    "SHRI. SUBHASH PHAL DESSAI",
    "SHRI. RAMESH TAWADKAR"
    };
    private string MLALeaders => string.Join("\n\n", MLA_Array);
    private string Secretary = "Smt. Namrata Ulman";
    private string[] RTI_Array = 
    {
        "SHRI MOHAN GAONKAR",
        "SMT N. ULMAN"
    };
    private string RTIOfficers => string.Join("\n\n", RTI_Array);

    public void SetDetails()
    {
        int menuChoice = 100;
        while (menuChoice != 0)
        {
            Console.WriteLine();
            Console.WriteLine("=============================== *** GOA LEGISLATURE DETAILS *** ===============================");
            Console.WriteLine("=============================== *** ENTER YOUR CHOICE *** ===============================");
            Console.WriteLine();
            Console.WriteLine("1] SPEAKER \t\t\t 2] DUTY SPEAKER \t\t\t 3] OPPOSITION LEADER");
            Console.WriteLine();
            Console.WriteLine("4] LIST OF MLA'S \t\t\t 5] SECRETARY \t\t\t 6] RTI OFFICERS");
            Console.WriteLine();
            Console.WriteLine("=============================== *** ENTER 0 TO RETURN TO MAIN MENU *** ===============================");
            Console.WriteLine();
            PrintAssemblyYear();
            Console.Write("=> ");
            menuChoice = int.Parse(Console.ReadLine());

            switch (menuChoice)
            {
                case 1:
                    Console.WriteLine($"SPEAKER FOR THE ASSEMBLY IS {Speaker}");
                    break;
                case 2:
                    Console.WriteLine($"DUTY SPEAKER FOR THE ASSEMBLY IS {DutySpeaker}");
                    break;
                case 3:
                    Console.WriteLine($"OPPOSITION LEADER IS {OppLeader}");
                    break;
                case 4:
                    Console.WriteLine($"----LIST OF MLA'S---- \n\n{MLALeaders}");
                    break;
                case 5:
                    Console.WriteLine($"SECRETARY IS {Secretary}");
                    break;
                case 6:
                    Console.WriteLine($"----RTI OFFICERS ARE---- \n\n{RTIOfficers}");
                    break;
                case 0:
                    return;
                default:
                    Console.WriteLine("PLEASE ENTER VALID INPUT");
                    break;
            }
        }
    }
}

File: Python/legislature.py
from utility import Utility
from gla import Gla
import sys

class Legislature(Gla, Utility):
    def __init__(self):
        super().__init__()
        self.speaker = "Shri. Ramesh Tawadkar"
        self.duty_speaker = "Shri. Joshua De Souza"
        self.oppo_leader = "Shri. Yuri Alemao"
        self.mla_array = [
            "SHRI. JIT AROLKAR",
            "SHRI. PRAVIN ARLEKAR",
            # ... (all other MLAs)
            "SHRI. RAMESH TAWADKAR"
        ]
        self.mla_leaders = "\n\n".join(self.mla_array)
        self.secretary = "Smt. Namrata Ulman"
        self.rti_array = ["SHRI MOHAN GAONKAR", "SMT N. ULMAN"]
        self.rti_officers = "\n\n".join(self.rti_array)

    def set_details(self):
        menuchoice = 100
        while menuchoice != 0:
            print()
            print("=============================== *** GOA LEGISLATURE DETAILS *** ===============================")
            print("=============================== *** ENTER YOUR CHOICE *** ===============================")
            print()
            print("1] SPEAKER \t\t\t 2] DUTY SPEAKER \t\t\t 3] OPPOSITION LEADER")
            print()
            print("4] LIST OF MLA'S \t\t\t 5] SECRETARY \t\t\t 6] RTI OFFICERS")
            print()
            print("=============================== *** ENTER 0 TO EXIT *** ===============================")
            print()
            self.print_assembly_year()
            menuchoice = int(input("=> ").strip())

            if menuchoice == 1:
                print(f"SPEAKER FOR THE ASSEMBLY IS {self.speaker}")
            elif menuchoice == 2:
                print(f"DUTY SPEAKER FOR THE ASSEMBLY IS {self.duty_speaker}")
            elif menuchoice == 3:
                print(f"OPPOSITION LEADER IS {self.oppo_leader}")
            elif menuchoice == 4:
                print(f"----LIST OF MLA'S---- \n\n{self.mla_leaders}")
            elif menuchoice == 5:
                print(f"SECRETARY IS {self.secretary}")
            elif menuchoice == 6:
                print(f"----RTI OFFICERS ARE---- \n\n{self.rti_officers}")
            elif menuchoice == 0:
                sys.exit()
            else:
                print("PLEASE ENTER VALID INPUT")
